src/layout/Group/config.ts

```diff
@@ -42,4 +42,13 @@ export const Config = new CG.component({
         .setTitle('Children')
         .setDescription('Array of component IDs that should be displayed in the group'),
     ).onlyIn(Variant.External),
+  )
+  .addProperty(
+    new CG.prop(
+      'headingLevel',
+      new CG.enum(1, 2, 3, 4, 5, 6)
```

Comment:
should we even allow using an h1 since the app-title already uses that? [It is not considered best practice to have multiple h1 elements on the same page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#avoid_using_multiple_h1_elements_on_one_page).

Reply:
Ah that's a good point! I will fix it!